tls/client-tls-pkcs12.c

```diff
@@ -131,6 +131,7 @@ int main(int argc, char** argv)
 
     /* Initialize wolfSSL */
     wolfSSL_Init();
+wolfSSL_Debugging_ON();
```

Comment:
Spacing?

Reply:
Oops...that slipped in there by accident.